elide-datastore/elide-datastore-multiplex/src/main/java/com/yahoo/elide/datastores/multiplex/MultiplexWriteTransaction.java

```diff
@@ -46,13 +48,13 @@ protected DataStoreTransaction beginTransaction(DataStore dataStore) {
     @Override
     public void save(Object entity, RequestScope requestScope) {
         getTransaction(entity).save(entity, requestScope);
-        dirtyObjects.add(this.multiplexManager.getSubManager(entity.getClass()), entity);
+        dirtyObjects.add(this.multiplexManager.getSubManager(getType(entity.getClass())), entity);
```

Comment:
Same comment with entity.getClass.  There should be no calls to getClass here.  Instead call getType from the entity dictionary on the entity itself.